lib/Twig/TokenStream.php

```diff
@@ -66,6 +68,9 @@ public function next()
     /**
      * Tests a token, sets the pointer to the next one and returns it or throws a syntax error.
      *
+     * @param int|string $primary
```

Comment:
how can it be a string ? It is passed to ``test()`` which expects an integer according to you phpdoc. One of them is wrong